section_to_course/models.py

```diff
@@ -0,0 +1,33 @@
+"""
+Database models for section_to_course.
+"""
+# from django.db import models
+from django.db import models
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.django.models import CourseKeyField, UsageKeyField
+
+
+class SectionToCourseLink(TimeStampedModel):
+    """
+    A model that links a section to a course.
+
+    .. no_pii:
+    """
+
+    source_course_id = CourseKeyField(max_length=255, db_index=True, null=False, blank=False)
+    destination_course_id = CourseKeyField(max_length=255, db_index=True, null=False, blank=False)
```

Comment:
@Agrendalath I think both of these fields are going to make more sense when we start writing the admin views and are going to need to look up the values by the source or destination course for some operations to avoid accidental duplication, for instance.